x-pack/plugins/security/server/plugin.ts

```diff
@@ -85,6 +86,10 @@ export interface SecurityPluginSetup {
    * Exposes services for audit logging.
    */
   audit: AuditServiceSetup;
+  /**
+   * Exposes services to access kibana roles per feature id with the GetDeprecationsContext
+   */
+  privilegeDeprecationsServicess: PrivilegeDeprecationsServices;
```

Comment:
Oops, I think this was a bad find/replace 😅  

Some places now have "privilege deprecations servicess" and others have "privilege deprecations services".
Whatever you pick, let's just make sure it's consistent.
I'd recommend "privilege deprecations service" everywhere:

```suggestion
  privilegeDeprecationsService: PrivilegeDeprecationsService;
```